tests/gnrc_dhcpv6_client_stateless/app.config

```diff
@@ -0,0 +1,2 @@
+CONFIG_GNRC_IPV6_NIB_ARSM=y
+CONFIG_GNRC_IPV6_NIB_SLAAC=y
```

Comment:
I now moved the former `CFLAGS` to the `app.config` like so. However, it seems as if the flags aren't actually necessary – in an older version I got a warning that both are needed to be set for using SLAAC but currently the test also works without them. So maybe the `app.config` could actually be removed.

Reply:
In case you do not want to remove it (IMHO, even if it is currently the default it might not be in the future, and at least for 6LoWPAN devices both config options _are_ actually deactivatet), you also need to add `CONFIG_KCONFIG_USEMODULE_GNRC_IPV6_NIB=y` for this to work. But Kconfig should warn you about this anyways when trying to compile ;-).

Reply:
Ah, I see, thank you for the explanation :) I now added `CONFIG_KCONFIG_USEMODULE_GNRC_IPV6_NIB=y`, keeping the config for the future sounds like a good idea :)